package.json

```diff
@@ -138,6 +138,7 @@
     "@spruceid/siwe-parser": "^1.1.3",
     "@truffle/codec": "^0.11.18",
     "@truffle/decoder": "^5.1.0",
+    "@weizman/snow": "^1.2.1",
```

Comment:
Are we going to change this to @lavamoat/snow eventually?

Reply:
f31c69e fix

Reply:
d6d47ca54 also